configcatclient/logger.py

```diff
@@ -1,26 +1,31 @@
 import logging
+import sys
 
 
-class Logger(object):
+class Logger(logging.LoggerAdapter):
     def __init__(self, name, hooks):
+        super(Logger, self).__init__(logging.getLogger(name), {})
         self._hooks = hooks
-        self._logger = logging.getLogger(name)
 
-    def debug(self, msg, *args, **kwargs):
-        self._logger.debug(msg, *args, **kwargs)
+    def process(self, msg, kwargs):
+        # Remove event ID from kwargs (as it's not a built-in argument expected by the logging framework)
+        # and put it in the extra dict so users can access it without parsing.
+        event_id = kwargs.pop('event_id', 0)
+        extra = kwargs.setdefault('extra', {})
+        extra['event_id'] = event_id
 
-    def info(self, msg, *args, **kwargs):
-        self._logger.info(msg, *args, **kwargs)
-
-    def warning(self, msg, *args, **kwargs):
-        self._logger.warning(msg, *args, **kwargs)
-
-    def warn(self, msg, *args, **kwargs):
-        self._logger.warning(msg, *args, **kwargs)
+        # Include the event ID in the message.
+        return ("[" + str(event_id) + "] " + msg, kwargs)
 
     def error(self, msg, *args, **kwargs):
-        self._hooks.invoke_on_error(msg)
-        self._logger.error(msg, *args, **kwargs)
+        self._hooks.invoke_on_error(Logger.format(msg, args))
+        super().error(msg, *args, **kwargs)
 
     def exception(self, msg, *args, **kwargs):
-        self.error(msg, *args, **kwargs)
+        self._hooks.invoke_on_error(Logger.format(msg, args, sys.exc_info()[1]))
+        super().exception(msg, *args, **kwargs)
+
+    @staticmethod
+    def format(msg, args, exc = None):
+        msg = msg % args if len(args) > 0 else msg
+        return msg if exc is None else msg + "\n" + str(exc)
```

Comment:
```suggestion
        return msg if exc is None else msg + '\n' + str(exc)
```

Reply:
Could you please change the format of `event_id = 1000` to `event_id=1000` (remove the extra spaces)